marshal_2_tinyint_test.go

```diff
@@ -0,0 +1,81 @@
+package gocql
```

Comment:
```suggestion
package gocql_test
```

Reply:
Impossible. It`s block access to `NativeType`, `CollectionType`...

Reply:
you still can do it, please address it.

Reply:
Ok